app/controllers/users_controller.rb

```diff
@@ -21,6 +21,8 @@ def create
       redirect_to @user, notice: 'User was successfully created.'
       NotifyMailer.api_key_confirmation(@user).deliver_now
       NotifyMailer.new_api_key_request(@user).deliver_now
+      analytics_update_service = AnalyticsUpdateService.new
```

Comment:
I wonder if we could move this to an ActiveJob? Then we can `perform_later` and run it async on the queue server.

Reply:
As discussed.

Reply:
Will this work? `AnalyticsUpdateService.new({@user[:api_key], @user[:department], @user[:service]}).add_new_service `

Reply:
I don't see any reason why not